core/services/fluxmonitorv2/flux_monitor.go

```diff
@@ -867,9 +868,10 @@ func (fm *FluxMonitor) pollIfEligible(pollReq PollRequestType, deviationChecker
 		"jobRun": map[string]interface{}{
 			"meta": metaDataForBridge,
 		},
+		"meta": metaDataForBridge,
```

Comment:
Note: a lot of jobs included the meta key under `jobSpec` meta already. I duplicated this to match the original behavior, but we can probably drop one

Reply:
We should drop one. Preferably the toplevel imo